packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -435,6 +444,12 @@ class CupertinoDatePicker extends StatefulWidget {
   /// Defaults to false.
   final bool showDayOfWeek;
 
+  /// Whether to show the time separator between hour and minute in
+  /// [CupertinoDatePickerMode.time] and [CupertinoDatePickerMode.dateAndTime] mode.
+  /// Throws an error if set to true in [CupertinoDatePickerMode.date] and [CupertinoDatePickerMode.monthYear] mode.
+  /// Defaults to false.
```

Comment:
Nit: I would try to make this text look like a paragraph with an 80 character line limit rather than starting each sentence on a new line.